db/seeds.rb

```diff
@@ -5,72 +5,297 @@
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "seasonal5.jpg", store: store_12)
+  CategoryItem.create(item: item, category: category_5)
+end
+
+puts "Adding items to the mattress category"
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "mattresses2.jpg", store: store_13)
+  CategoryItem.create(item: item, category: category_4)
+end
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "mattresses1.jpg", store: store_14)
+  CategoryItem.create(item: item, category: category_4)
+end
+
+puts "Adding items to the kitchen category"
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "kitchen5.jpg", store: store_15)
+  CategoryItem.create(item: item, category: category_3)
+end
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "kitchen4.jpg", store: store_16)
+  CategoryItem.create(item: item, category: category_3)
+end
+
+puts "Adding items to the furniture category"
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "furniture9.jpg", store: store_17)
+  CategoryItem.create(item: item, category: category_2)
+end
+
+25.times do |index|
+  item = Item.create!(name: Faker::Commerce.unique.product_name,
+               description: Faker::Lorem.sentence(20, false, 0).chop,
+               price: rand(100..1000), image_path: "furniture10.jpg", store: store_18)
+  CategoryItem.create(item: item, category: category_2)
+end
+
+  puts "Adding items to the bath category"
+
+  25.times do |index|
+    item = Item.create!(name: Faker::Commerce.unique.product_name,
+                 description: Faker::Lorem.sentence(20, false, 0).chop,
+                 price: rand(100..1000), image_path: "bath4.jpg", store: store_20)
+    CategoryItem.create(item: item, category: category_1)
+  end
+
+  25.times do |index|
+    item = Item.create!(name: Faker::Commerce.unique.product_name,
+                 description: Faker::Lorem.sentence(20, false, 0).chop,
+                 price: rand(100..1000), image_path: "bath3.jpg", store: store_19)
+    CategoryItem.create(item: item, category: category_1)
+  end
+
+  # -----------------------------Users
+1000.times do |users|
+  name = Faker::Name.unique.name
```

Comment:
We will need to update this now as my pr has updated user attributes. User now has email and phone as validated attributes to make 2 factor authentication functional